### DISCUSSION TITLE
Integrate Sonner

### DISCUSSION THREAD

**Discussion by Acetyld** [primefaces/community#2593] [PrimeVue]
Hi there!

I think it would be a nice addon to also add https://vue-sonner.vercel.app/ to the components (just like you did with wysiwyg editor and charts)

Current toast are limited, for example updating one by id is not possible.

Sonner is opinionated so added it besides toast is a good way to go i think, we currently got it integrated separately, but would be lovely to have a default primevue unstyled style on this =) So it takes over primevue style

**Comment by j0rgedev** [comment]
upvote
